### DISCUSSION TITLE
Update password : how to verify old password?

### DISCUSSION THREAD

**Discussion by vd3d** [supabase/supabase#28387] [Questions] [answered]
Hi there,

I'm currently able to change my password, but I also ask the user to enter his/her old password.

But, I didn't find any API to verify that the old password is correct. Is it possible to do ?

Thanks

**Comment by GaryAustin1** [comment, marked as answer]
There is nothing built in.  I've seen several discussions on doing this.

https://github.com/orgs/supabase/discussions/4042
https://github.com/orgs/supabase/discussions/11412
